type.proto

```diff
@@ -77,6 +77,14 @@ message Object {
 }
 
 message ProfilingResult {
+  message Characteristic {
+    oneof member_oneof {
+      uint64 size = 1;
```

Comment:
we are mostly using it for size values e.g. (number of points) but you can input any integer value (also negative)

Reply:
then, variable name `integer` is OK?
 ```
message Characteristic {
    oneof member_oneof {
      int64 integer = 1;

Reply:
Puuh. `integer` sounds to me not like a name should give a variable. Maybe `integer_number` and `decimal_number`?

Reply:
OK, I will go with that.